subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/server/http/BlockingHttpsServer.java

```diff
@@ -67,6 +68,10 @@ private String[] stripFilteredProtocols(String[] allProtocols) {
     }
 
     public void configure(TestKeyStore testKeyStore) {
-        configure(testKeyStore, __ -> true);
+        configure(testKeyStore, false, __ -> true);
```

Comment:
🧐 Double underscore as an identifier seems non-idiomatic in Java 8.

Reply:
It was there before, but I'll fix that

Reply:
`Use of '_' as a lambda parameter name is not allowed`. 
Apparently, there is a couple of similar usages in our code base. 
I don't think named argument would be better here.